bin/hdfcoinc/pycbc_fit_sngls_split_binned

```diff
@@ -153,16 +154,25 @@ if args.max_bin_param:
     pbin_upper_lim = float(args.max_bin_param)
 else:
     pbin_upper_lim = params[args.bin_param].max()
-bb_input = (params[args.bin_param].min(), pbin_upper_lim, args.num_bins)
+
+
+if args.bin_param == 'template_duration' and params[args.bin_param].min() == 0:
+    logging.warn('WARNING: Some templates do not contain triggers')
+    pbin_lower_lim = params[args.bin_param][params[args.bin_param] > 0].min()
```

Comment:
comment this line to tell us what it does

Reply:
Ditto as above: Comment added, hopefully it is clear